source/extensions/filters/network/thrift_proxy/conn_manager.h

```diff
@@ -272,6 +273,11 @@ class ConnectionManager : public Network::ReadFilter,
   bool stopped_{false};
   bool half_closed_{false};
   TimeSource& time_source_;
+
+  // The number of requests accumulated on the current connection. A connection is processed by only
+  // one thread, so there is no need to consider the thread safety of the count.
```

Comment:
I _think_ we can drop the 2nd sentence since it's a common assumption that conn managers are per thread.

Reply:
:ok_hand: